src/components/ReactGoogleMaps/ReactGoogleMaps.jsx

```diff
@@ -263,7 +268,28 @@ const ReactGoogleMaps = () => {
               onMarkerClick(resource);
             }}
             position={{ lat: resource.latitude, lng: resource.longitude }}
-            icon={{ url: phlaskMarkerIconV2(resource.resource_type, 56, 56) }}
+            icon={{
+              url: (() => {
+                if (
+                  selectedPlace?.latitude === resource.latitude &&
+                  selectedPlace?.longitude === resource.longitude
+                ) {
+                  switch (resource.resource_type) {
+                    case 'WATER':
+                      return PinWaterActive();
+                    case 'FOOD':
+                      return PinFoodActive();
+                    case 'FORAGE':
+                      return PinForagingActive();
+                    case 'BATHROOM':
+                      return PinBathroomActive();
```

Comment:
@icycoldveins , could you consolidate the Pin*Active files into a single file, similar to PhlaskMarkerIconV2.js and how it handles the resource type as an input parameter? That would help with consistency in this section